### PR TITLE
Openpilot webcam support fixed

### PR DESCRIPTION
**Pull request by MikeBusuttil**
- `USE_WEBCAM` toggle fixed
- webcam [readme](https://github.com/commaai/openpilot/blob/7576f3e9adef9ddd7e133a7067704b8b656c6d56/tools/webcam/README.md) instructions updated
- environment variables (optionally) used for camera specifying

Thanks to @jyoung8607 for all the help getting this going

### CODE REVIEW

File: tools/webcam/README.md [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
 ## GO
 ```
-cd ~/openpilot/system/manager
-NOSENSOR=1 USE_WEBCAM=1 ./manager.py
+USE_WEBCAM=1 system/manager/manager.py
 ```
-- Start the car, then the UI should show the road webcam's view
-- Adjust and secure the webcams (you can run tools/webcam/front_mount_helper.py to help mount the driver camera)
+- Start the car, then the UI should show the road webcam's view (TODO: should it!? This needs fixing then)
+- Adjust and secure the webcams.
```
Comment (MikeBusuttil) [inline tools/webcam/README.md:27]: The file referenced here can be restored with `git checkout c4f322ce2de191a07db992e649f73c128048bb74^ -- tools/webcam/front_mount_helper.py`
I couldn't get it to work for me on my janky WSL setup so I left it out.  Will test on native Ubuntu to verify

```
#!/usr/bin/env python
import numpy as np

# copied from openpilot.common.transformations/camera.py
eon_dcam_focal_length = 860.0  # pixels
webcam_focal_length = 908.0  # pixels

eon_dcam_intrinsics = np.array([
  [eon_dcam_focal_length,   0,   1152/2.],
  [  0,  eon_dcam_focal_length,  864/2.],
  [  0,    0,     1]])

webcam_intrinsics = np.array([
  [webcam_focal_length,   0.,   1280/2.],
  [  0.,  webcam_focal_length,  720/2.],
  [  0.,    0.,     1.]])

cam_id = 6

if __name__ == "__main__":
  import cv2  # pylint: disable=import-error

  trans_webcam_to_eon_front = np.dot(eon_dcam_intrinsics, np.linalg.inv(webcam_intrinsics))

  cap = cv2.VideoCapture(cam_id)
  cap.set(cv2.CAP_PROP_FRAME_WIDTH, 1280)
  cap.set(cv2.CAP_PROP_FRAME_HEIGHT, 720)

  while (True):
    print('hi')
    ret, img = cap.read()
    if ret:
      img = cv2.warpPerspective(img, trans_webcam_to_eon_front, (1152, 864), borderMode=cv2.BORDER_CONSTANT, borderValue=0)
      img = img[:, -864//2:, :]
      cv2.imshow('preview', img)
      cv2.waitKey(10)
```

**Comment by MikeBusuttil** [comment]
[This](https://github.com/commaai/openpilot/blob/7576f3e9adef9ddd7e133a7067704b8b656c6d56/tools/webcam/README.md?plain=1#L26) is untested:
- Start the car, then the UI should show the road webcam's view
Does anyone have a PC they can hook up to a car to verify?

File: system/manager/process_config.py [hunk 2]
```diff
@@ -63,14 +63,15 @@ def and_(*fns):
...
+  NativeProcess("camerad", "system/camerad", ["./camerad"], driverview, enabled=not WEBCAM),
+  PythonProcess("camerad", "tools.webcam.camerad", driverview, enabled=WEBCAM),
   NativeProcess("logcatd", "system/logcatd", ["./logcatd"], only_onroad),
   NativeProcess("proclogd", "system/proclogd", ["./proclogd"], only_onroad),
   PythonProcess("logmessaged", "system.logmessaged", always_run),
   PythonProcess("micd", "system.micd", iscar),
   PythonProcess("timed", "system.timed", always_run, enabled=not PC),
 
-  NativeProcess("dmonitoringmodeld", "selfdrive/modeld", ["./dmonitoringmodeld"], driverview, enabled=(not PC or WEBCAM)),
+  NativeProcess("dmonitoringmodeld", "selfdrive/modeld", ["./dmonitoringmodeld"], driverview, enabled=(WEBCAM or not PC)),
```
Comment (MikeBusuttil) [inline system/manager/process_config.py:74]: just for readability

File: tools/webcam/README.md (hunk 1 above)
Comment (MikeBusuttil) [reply tools/webcam/README.md:27]: GREAT idea!  Will work on a follow-up PR with easy-to-follow user instructions

File: system/manager/process_config.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 from openpilot.system.hardware import PC, TICI
 from openpilot.system.manager.process import PythonProcess, NativeProcess, DaemonProcess
 
-WEBCAM = os.getenv("USE_WEBCAM") is not None
+WEBCAM = os.getenv("USE_WEBCAM") or os.getenv("ROAD_CAM") or os.getenv("DRIVER_CAM")
```
Comment (adeebshihadeh) [inline system/manager/process_config.py:9]: revert this

File: system/manager/process_config.py [hunk 4]
```diff
@@ -63,14 +63,15 @@ def and_(*fns):
 procs = [
   DaemonProcess("manage_athenad", "system.athena.manage_athenad", "AthenadPid"),
 
-  NativeProcess("camerad", "system/camerad", ["./camerad"], driverview),
+  NativeProcess("camerad", "system/camerad", ["./camerad"], driverview, enabled=not WEBCAM),
+  PythonProcess("camerad", "tools.webcam.camerad", driverview, enabled=WEBCAM),
```
Comment (adeebshihadeh) [inline system/manager/process_config.py:67]:
```suggestion
  PythonProcess("webcamerad", "tools.webcam.camerad", driverview, enabled=WEBCAM),
```

File: system/manager/process_config.py (hunk 3 above)
Comment (MikeBusuttil) [reply system/manager/process_config.py]: ✅done

File: system/manager/process_config.py (hunk 4 above)
Comment (MikeBusuttil) [reply system/manager/process_config.py]: ✅done

As far as I know, nothing relies on the hard-coded name `camerad`

**Pull request merged**